libshaderc/include/shaderc.h

```diff
@@ -105,6 +111,11 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set flags for the rules on which warnings or errors will be issued. This can
+// either be an individual flag, or a set of flags or'd together
+void shaderc_compile_options_set_message_rules(
+    shaderc_compile_options_t options, shaderc_msg_rules rules);
+
```

Comment:
No strong opinion on this. We don't have a getter for shaderc_compile_options_add_macro_definition(), so I followed the same approach. I don't have a problem with adding one if that's desired.